Source/WebCore/svg/SVGGeometryElement.cpp

```diff
@@ -74,7 +74,10 @@ ExceptionOr<Ref<SVGPoint>> SVGGeometryElement::getPointAtLength(float distance)
         return Exception { ExceptionCode::InvalidStateError };
 
     // Spec: Clamp distance to [0, length].
-    distance = clampTo<float>(distance, 0, getTotalLength());
+    auto totalLength =  getTotalLength();
+    if (totalLength.hasException())
+        return totalLength.releaseException();
+    distance = clampTo<float>(distance, 0, totalLength.releaseReturnValue());
```

Comment:
`totalLength` is just float so it does not need the move semantic. So I think we can use `totalLength.returnValue()`.

Reply:
Done and also updated it for `SVGPathElement`.